nixos/modules/installer/tools/tools.nix

```diff
@@ -166,7 +166,7 @@ in
         # Enable CUPS to print documents.
         # services.printing.enable = true;
 
-        # Enable sound.
+        # Enable sound (may be already enabled by a desktop manager like gnome).
```

Comment:
I think it might be sensible to only include this section when GNOME is not used (apparently Plasma does not automatically enable pulseaudio – Pantheon, Cinnamon and Plasma mobile do, but they do not have installer support).